README.md

```diff
@@ -1,2 +1,179 @@
-# stellar-docs
-Documentation for Stellar
+# Stellar Documentation and API Reference
+
+Welcome to the official home repository for [Documentation][docs] and
+[API Reference][api] for the [Stellar Network][stellar].
+
+# Table of Contents
+
+- [Stellar Documentation and API Reference](#stellar-documentation-and-api-reference)
+- [Table of Contents](#table-of-contents)
+- [Contributing](#contributing)
+- [Quick Start](#quick-start)
+- [Repository Structure](#repository-structure)
+- [Using Markdown](#using-markdown)
+  - [Markdown Basics](#markdown-basics)
+  - [Custom Markdown](#custom-markdown)
+    - [Alert](#alert)
+    - [Code Example](#code-example)
+
+# Contributing
+
+Contributions are more than welcome! Thank you! 🎉
+
+Before diving in, please read our [Stellar Contribution Guide][contrib] for
+details on contributing to Stellar's various repositories. Please take special
+note of the code of conduct and contributor license agreement.
```

Comment:
Should we link to the Code of Conduct (https://github.com/stellar/.github/blob/master/CODE_OF_CONDUCT.md) and Contributor License Agreement (I actually didn't see this link in the Contribution Guide) here?

Reply:
Good catch! Looks like the CLA was removed from the process... last... January... 🤦🏻 Should've done a bit more due diligence on that one 🤣